email-verification/main.go

```
@@ -0,0 +1,32 @@
+	// Services
+	verificationService := application.NewVerificationService(repo, emailService, codeGen, config.GetConfig())
+
+	// Handlers
+	verificationHandler := handlers.NewVerificationHandler(verificationService)
+
+	// Router
+	app := fiber.New()
+	app.Post("/verify/send/:email", verificationHandler.SendVerification)
+	app.Post("/verify/check/:email/:code", verificationHandler.CheckVerification)
+
+	app.Listen(":3000")
```

golangci-lint (1.62.2)

Error return value of `app.Listen` is not checked

(errcheck)